docker/readme.md

```diff
@@ -4,10 +4,11 @@
 ```shell
 docker build --build-arg UBUNTU=xenial --build-arg BUILD_TYPE=Release -t seahorn/seahorn-build:xenial -f docker/seahorn-build.Dockerfile .
 ```
-The above generates a container with all dependencies required to build SeaHorn, other than SeaHorn itself.
+The above generates a container with all dependencies required to build SeaHorn,
```

Comment:
this file should not be changing... maybe something is wrong with rebase and this PR pulled in some other changes from the master?

Reply:
That appears to be the case. It looks like some other files have changed after rebasing.

Reply:
will you be able to fix this PR or do you need help with this?